packages/ClearButton/index.tsx

```diff
@@ -10,13 +10,11 @@ export type ClearButtonProps = CreateWuiProps<
   Omit<ButtonOptions, 'shape' | 'title' | 'variant'>
 >
 
-export const ClearButton = forwardRef<'button', ClearButtonProps>(
-  ({ size = 'xs', ...rest }, ref) => (
-    <S.ClearButton ref={ref} shape="circle" size={size} title="Clear" variant="tertiary" {...rest}>
-      <CrossIcon size={size} />
-    </S.ClearButton>
-  )
-)
+export const ClearButton = forwardRef<'button', ClearButtonProps>(({ ...rest }, ref) => (
+  <S.ClearButton p="xxs" ref={ref} shape="circle" title="Clear" variant="ghost" {...rest}>
+    <CrossIcon size="xs" />
```

Comment:
Is it normal to always have a xs CrossIcon? The width and the height set for the ClearButton in the style file override the size props?

Reply:
Since the new rework of the button you don't need to specify a size for the icon, it will fit the size of the button perfectly. (You can check here -> https://feat-forms-new-spacing-colors--welcome-ui.netlify.app/components/button)

Reply:
Thanks for the explanation @Leiksa 🙏